controllers/dbaas.redhat.com/configmap.go

```diff
@@ -45,9 +45,10 @@ const (
 
 var labels = map[string]string{RELATEDTOLABELNAME: RELATEDTOLABELVALUE, TYPELABELNAME: TYPELABELVALUE}
 
+// CreateBridgeRegistrationConfigMap invoked from main.go prior to completion of manager start-up so controller/cache not ready, using manager to get  client & API Reader instead."
 func (r *CrunchyBridgeInventoryReconciler) CreateBridgeRegistrationConfigMap(mgr manager.Manager) error {
```

Comment:
Should we check if the dbaas-operator namespace exists before creating the configmap? What if the namespace does not exist, should the operator continue to run, or should it terminate?

Reply:
if there is no namespace then, operator will print error inside the logs, will say no namespace exists, and will not terminate.